<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
    <widget name="Form" class="QGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
        <property name="title">
            <string>Properties</string>
        </property>
        <property name="alignment">
            <set>Qt::AlignCenter</set>
        </property>
        <layout name="verticalLayout" class="QVBoxLayout">
            <property name="spacing">
                <number>12</number>
            </property>
            <item>
                <widget name="title_label" class="QLabel">
                    <property name="styleSheet">
                        <string notr="true">font: 22pt;</string>
                    </property>
                    <property name="text">
                        <string />
                    </property>
                </widget>
            </item>
            <item>
                <widget name="properties_info_label" class="QLabel">
                    <property name="styleSheet">
                        <string notr="true">font: 15pt;
                        </string>
                    </property>
                    <property name="text">
                        <string />
                    </property>
                </widget>
            </item>
            <item>
                <layout name="properties_layout" class="QFormLayout">
                    <property name="horizontalSpacing">
                        <number>100</number>
                    </property>
                    <property name="verticalSpacing">
                        <number>12</number>
                    </property>
                </layout>
            </item>
            <item>
                <widget name="other_properties_info_label" class="QLabel">
                    <property name="styleSheet">
                        <string notr="true">font:15pt;</string>
                    </property>
                    <property name="text">
                        <string />
                    </property>
                </widget>
            </item>
            <item>
                <layout name="other_properties_layout" class="QFormLayout">
                    <property name="horizontalSpacing">
                        <number>100</number>
                    </property>
                    <property name="verticalSpacing">
                        <number>12</number>
                    </property>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
